lib/archethic/shared_secrets/node_renewal_scheduler.ex

```diff
@@ -117,9 +119,26 @@ defmodule Archethic.SharedSecrets.NodeRenewalScheduler do
       "Node shared secrets will be renewed in #{Utils.remaining_seconds_from_timer(timer)}"
     )
 
-    if NodeRenewal.initiator?() do
+    tx =
+      NodeRenewal.next_authorized_node_public_keys()
+      |> NodeRenewal.new_node_shared_secrets_transaction(
+        :crypto.strong_rand_bytes(32),
+        :crypto.strong_rand_bytes(32)
+      )
+
+    validation_nodes = get_validation_nodes()
+
+    if trigger_node?(validation_nodes) do
       Logger.info("Node shared secrets renewal creation...")
-      make_renewal()
+      make_renewal(tx)
```

Comment:
I think you can pass the `validation_nodes` in parameter as so the `Archethic.send_new_transaction` will not recalculate the validation nodes

Reply:
ok, will do that anything else